tensoevent/tensoevent.go

```diff
@@ -79,6 +99,8 @@ const (
 	OutcomeNotDeployed Outcome = "not-deployed"
 	//OutcomeSucceeded describes a Helm release that succeeded.
 	OutcomeSucceeded Outcome = "succeeded"
+	//OutcomeFailed describes a failed release
+	OutcomeFailed Outcome = "failed"
```

Comment:
To avoid confusion with when to use this outcome vs. more specific outcomes like `helm-upgrade-failed`, I'd like to be clearer here, e.g.
```suggestion
	//OutcomeTerraformRunFailed describes a Terraform run that failed.
	OutcomeTerraformRunFailed Outcome = "terraform-run-failed"
```